Add Telemetry devices reference (#5164)
opensearch-project/documentation-website: _benchmark/user-guide/telemetry.md

---
layout: default
title: Enabling telemetry devices
nav_order: 15
parent: User guide
---

> Telemetry results will not show up in the summary report. The only way to visualize telemetry results is to ingest the data into OpenSearch and visualize the data in OpenSearch Dashboards. 

Reviewer: ```suggestion
Telemetry results will not appear in the summary report. To visualize telemetry results, ingest the data into OpenSearch and visualize the data in OpenSearch Dashboards. 
```
